src/signTxPoolRegistration.h

```diff
@@ -12,54 +12,84 @@
 // SIGN_STAGE_CERTIFICATES = 28
 // CERTIFICATE_TYPE_STAKE_POOL_REGISTRATION = 3
 typedef enum {
-	STAKE_POOL_REGISTRATION_PARAMS = 2830,
-	STAKE_POOL_REGISTRATION_OWNERS = 2831,
-	STAKE_POOL_REGISTRATION_RELAYS = 2832,
-	STAKE_POOL_REGISTRATION_METADATA = 2833,
-	STAKE_POOL_REGISTRATION_CONFIRM = 2834,
-	STAKE_POOL_REGISTRATION_FINISHED = 2835
+	STAKE_POOL_REGISTRATION_INIT = 2830,
+	STAKE_POOL_REGISTRATION_POOL_KEY = 2831,
+	STAKE_POOL_REGISTRATION_VRF_KEY = 2832,
+	STAKE_POOL_REGISTRATION_FINANCIALS = 2833,
+	STAKE_POOL_REGISTRATION_REWARD_ACCOUNT = 2834,
+	STAKE_POOL_REGISTRATION_OWNERS = 2835,
+	STAKE_POOL_REGISTRATION_RELAYS = 2836,
+	STAKE_POOL_REGISTRATION_METADATA = 2837,
+	STAKE_POOL_REGISTRATION_CONFIRM = 2838,
+	STAKE_POOL_REGISTRATION_FINISHED = 2839,
 } sign_tx_pool_registration_state_t;
 
-enum {
-	SIGN_TX_POOL_OWNER_TYPE_PATH = 1,
-	SIGN_TX_POOL_OWNER_TYPE_KEY_HASH = 2,
-};
+typedef struct {
+	data_description_kind_t descriptionKind;
```

Comment:
data description kind seems too generic to me - isn't this rather like "key reference type" - one being the key hash, the there key derivation path?

Reply:
fair point and as long as "kind" is the standard suffix in the codebase I'm not against it, though it seems to me that it currently isn't, and switching ad-hoc to it may bring more confusion within the codebase when adding new features ("I see both instances of "type" and "kind" and I'm not sure how they differ" dilemma when naming new things) than the potential initial confusion of somebody coming from another codebase/language.

Also, what comes before that "type/kind" suffix should already be clear enough to be searchable on its own otherwise that can be a sign the name is too generic/misleading